### PR TITLE
Added Products Page

### PR DESCRIPTION
**Pull request by SimranjitKaurMaan**
Added Products Page

image

### CODE REVIEW

File: products-vertical.html [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+                      </div>
+                    </div>
+                </aside>
+                <main>
+                    <div class="main-section">
+                        <div class="main-container flex-row-wrap-center">
+                            <div class="card-container card-icon-overlay-container">
+                                <span class="card-wish-icon"><i class="fas fa-heart icon-unchecked"></i></span>
+                                <div class="card-body card-vertical-body">
+                                    <img class="card-img card-vertical-img" src="./assets/avatar-2.jpeg" sizes="(min-width: 600px) 200px,100px"/>
```
Comment (shubhamkumar648) [inline products-vertical.html]: here also forgot to put alt attribute in img tag

File: products-vertical.html [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+                                    </div>
+                                </div>
+                                <div class="card-footer">
+                                   <button class="btn btn-primary">Add to Cart</button>
+                                </div>
+                            </div>
+                            <div class="card-container card-icon-overlay-container">
+                                <span class="card-wish-icon"><i class="fas fa-heart icon-unchecked"></i></span>
+                                <div class="card-body card-vertical-body">
+                                    <img class="card-img card-vertical-img" src="./assets/avatar-2.jpeg" sizes="(min-width: 600px) 200px,100px"/>
```
Comment (shubhamkumar648) [inline products-vertical.html]: same here in img tag

**Review by shubhamkumar648 — APPROVED** [review]
Add your alt tag so that if the image can not display then  alt attributes provide alternative information

**Review by srejitk — APPROVED** [review]
Looks good. One  small suggestion would be to use a larger size of Like/Fav button. And consider adding inline comments in HTML to separate sections.

**Review by ttahm3d — APPROVED** [review]
Looks good, Can be merged.

File: products-vertical.html (hunk 2 above)
Comment (SimranjitKaurMaan) [reply products-vertical.html]: Thanks for pointing out. I have added the alt tags.

**Comment by SimranjitKaurMaan** [comment]
> Looks good. One small suggestion would be to use a larger size of Like/Fav button. And consider adding inline comments in HTML to separate sections.

Sure. I have added comments in html pages also. Also, I am planning to refactor the cards so like/fav buttons wont be required. Will remove them.

**Comment by SimranjitKaurMaan** [comment]
> Looks good, Can be merged.

Thanks for reviewing.

**Pull request merged**